src/app/api/services/dot-themes/dot-themes.service.ts

```diff
@@ -26,7 +26,6 @@ export class DotThemesService {
                 method: RequestMethod.Get,
                 url: 'v1/themes/id/' + inode
             })
-            .pluck('entity')
-            .map((themes: DotTheme[]) => themes[0]);
```

Comment:
Why? and you didn't update what this method return, now returns an array of ONE item... make no sense to me.